src/app/libp2p_helper/src/libp2p_helper/app.go

```diff
@@ -325,13 +354,13 @@ func handleStreamReads(app *app, stream net.Stream, idx uint64) {
 		app.writeMsg(mkStreamCompleteUpcall(idx))
 		err := stream.Close()
 		if err != nil {
-			app.P2p.Logger.Warning("Error while closing the stream: ", err.Error())
+			app.P2p.Logger.Warn("Error while closing the stream: ", err.Error())
 		}
 	}()
 }
 
 func beginMDNS(app *app, foundPeerCh chan peerDiscovery) error {
-	mdns, err := mdns.NewMdnsService(app.Ctx, app.P2p.Host, time.Minute, "_coda-discovery._udp.local")
+	mdns, err := mdns.NewMdnsService(app.Ctx, app.P2p.Host, time.Second, "_coda-discovery._udp.local")
```

Comment:
Isn't 1 second a bit aggressive for this interval?

Reply:
this was an accidental change